synapse/storage/schema/main/delta/72/01event_txn_device_id.sql.postgres

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2022 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
more comments please. What is this delta doing? why?

(see https://github.com/matrix-org/synapse/blob/ba03e66845845cc2afd37e3f8843707ca29f2b22/synapse/storage/schema/main/delta/71/01rebuild_event_edges.sql.postgres#L16 for example, though I now wonder why I used `--` comments rather than a `/*` one.